features/works/work_create.feature

```diff
@@ -292,3 +292,17 @@ Feature: Create Works
       And I press "Post Without Preview"
     Then I should see "Publication date can't be in the future."
     When I jump in our Delorean and return to the present
+
+  Scenario: Adding a coauthor to a work adds the coauthor to all existing chapters.
```

Comment:
Thanks for adding this! Just a quick note about formatting cukes: we try to make it so there aren't multiple steps beginning with `Given`, `When`, or `Then` in a row, and only `And` or `But` steps are indented. So something more like:

```
 Scenario: Adding a coauthor to a work adds the coauthor to all existing chapters.
    Given the following activated users exists
    | login             | password   | email                     |
    | author            | password   | author@example.com        |
    | coauthor          | password   | coauthor@example.com      |
    When I am logged in as "author"
      And I post the chaptered work "Chaptered Work"
      And I add the co-author "coauthor" to the work "Chaptered Work"
    Then I should see "author, coauthor" within ".byline"
    When I follow "Next Chapter →"
    Then I should see "Chapter 2"
      And I should see "author, coauthor" within ".byline"
```

Also, I think newer versions of FactoryGirl get rid of support for tables, so for a bit of future-proofing, it might be better to use this step instead:

```
Given the user "author" exists and is activated
  And the user "coauthor" exists and is activated
```